### ISSUE TITLE
dojo: unlinking a %sole-enabled app makes dojo unusable

### ISSUE THREAD

**Issue by yosoyubik** [urbit/urbit#1908]
This issue happened after moving the expression (`=^  edit  state.consol.sat  (to-sole set+~)`) to a different arm in my app. 

```
    ::  I got this message [%receive-sync [%his 4 4] %own 4 3]
    ::  when running this expression. It also unlinked the app from %sole
    ::
    =^  inv  state.consol.sat  (~(transceive sole state.consol.sat) +.act)
```

```
    ::   This worked before but after I moved it to a different arm, caused the error
    ::
    =^  edit  state.consol.sat  (to-sole set+~)
    ::  Added for reference of what to-sole is doing
    ::
    ++  to-sole
      |=  inv=sole-edit
      ^-  [sole-change sole-share]
      (~(transmit ..transmit state.consol.sat) inv)
```

After a new edit was sent ("typed command and pressed enter") to clear the prompt, running `transceive` again ("started typing a command") caused the aforementioned error, I assume because of different clocks in sole-state and sole-change, which I guess means that the state was not updated properly after running `+to-sole`. After that I linked the app again to the console and, after sending a new edit, it showed couple of `%ap-lame`, `%drum-unauthorized` and then a million malloc error. 

<details><summary>Full error output</summary>
<p>
<pre><code>
[%receive-sync [%his 4 4] %own 4 3]
[unlinked from [p=~zod q=%week0]]
/~zod/home/0/app/week0/hoon:<[259 7].[289 9]>
/~zod/home/0/app/week0/hoon:<[260 7].[289 9]>
/~zod/home/0/app/week0/hoon:<[261 7].[289 9]>
/~zod/home/0/app/week0/hoon:<[287 9].[288 17]>
/~zod/home/0/app/week0/hoon:<[287 36].[287 79]>
/~zod/home/0/lib/sole/hoon:<[129 3].[132 34]>
/~zod/home/0/lib/sole/hoon:<[130 3].[132 34]>
/~zod/home/0/lib/sole/hoon:<[131 3].[132 34]>
/~zod/home/0/lib/sole/hoon:<[131 20].[131 34]>
/~zod/home/0/lib/sole/hoon:<[100 3].[110 45]>
/~zod/home/0/lib/sole/hoon:<[101 3].[110 45]>
/~zod/home/0/lib/sole/hoon:<[102 5].[103 7]>
/~zod/home/0/lib/sole/hoon:<[103 5].[103 7]>
[%drum-coup-fail ~zod 1 p=~zod q=%week0]
'tile'
"sending note(s)"
> |link %week0
>=
[linked to [p=~zod q=%week0]]
"sending note(s)"
 Note added to your folder
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 3].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 16].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 20].[164 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %diff-sole-effect-drum-phat]
]
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 3].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 16].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 20].[164 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %diff-sole-effect-drum-phat]
]
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 3].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 16].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 20].[164 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %diff-sole-effect-drum-phat]
]
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[228 3].[234 42]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[228 16].[234 42]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[228 20].[228 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %coup-drum-phat]
]
> |unlink %week0
crud: %reap event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[171 3].[174 49]>
[%drum-unauthorized [%our ~zod] %src ~zod]
/~zod/home/0/lib/hood/drum/hoon:<[172 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 16].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 20].[173 27]>
/~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
~zod:dojo>
crud: %reap event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[171 3].[174 49]>
[%drum-unauthorized [%our ~zod] %src ~zod]
/~zod/home/0/lib/hood/drum/hoon:<[172 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 16].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 20].[173 27]>
/~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
~zod:dojo> urbit(88932,0x114d445c0) malloc: *** error for object 0x7ff1db42e500: pointer being freed was not allocated
urbit(88932,0x114d445c0) malloc: *** set a breakpoint in malloc_error_break to debug
</pre></code>
</p>
</details>

After it crashed and I booted the ship up again, dojo didn't accept any command, showing error after each key press, and could only be exited with ctrl+z.

<details><summary>Error after booting again</summary>
<p>
<pre><code>
crud: %coup event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[188 3].[189 16]>
/~zod/home/0/lib/hood/drum/hoon:<[188 16].[189 16]>
/~zod/home/0/lib/hood/drum/hoon:<[188 20].[188 27]>
/~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
crud: %coup event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[188 3].[189 16]>
/~zod/home/0/lib/hood/drum/hoon:<[188 16].[189 16]>
/~zod/home/0/lib/hood/drum/hoon:<[188 20].[188 27]>
/~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
crud: %coup event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[188 3].[189 16]>
/~zod/home/0/lib/hood/drum/hoon:<[188 16].[189 16]>
/~zod/home/0/lib/hood/drum/hoon:<[189 3].[189 16]>
/~zod/home/0/lib/hood/drum/hoon:<[358 3].[369 40]>
/~zod/home/0/lib/hood/drum/hoon:<[359 3].[369 40]>
/~zod/home/0/lib/hood/drum/hoon:<[366 3].[369 40]>
/~zod/home/0/lib/hood/drum/hoon:<[366 7].[366 18]>
/~zod/home/0/lib/hood/drum/hoon:<[366 11].[366 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
</p>
</pre></code>
</details>

I'm sure that this is me doing something very wrong with %sole, but the fact that it crashed in this way and apparently broke the console made me think that something might be amiss here.

Probably this is hard to understand, described with words that is, so I can make a small video of what happened if it's difficult to replicate.

PS: And there I was, quickly finishing the assignment for hoonschool before starting with the final project...

This is the app to blame:

https://gist.github.com/yosoyubik/b81362858f6f32c06a68ae317965baef

**Comment by yosoyubik** [comment]
FYI, this version works and doesn't crash:
https://gist.github.com/yosoyubik/7f9ff48a17a0e5b4982fee93c1160d5a
but unlinking crashes and makes dojo unusable as well.

<details> <summary>error output</summary>
<p>
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 3].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 16].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 20].[164 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %diff-sole-effect-drum-phat]
]
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 3].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 16].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 20].[164 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %diff-sole-effect-drum-phat]
]
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 3].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 16].[167 20]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[164 20].[164 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %diff-sole-effect-drum-phat]
]
[ ! /~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
  ! /~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[228 3].[234 42]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[228 16].[234 42]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[228 20].[228 27]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
  ! /~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
  [%ap-lame %hood %coup-drum-phat]
]
> |unlink %week0
crud: %reap event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[171 3].[174 49]>
[%drum-unauthorized [%our ~zod] %src ~zod]
/~zod/home/0/lib/hood/drum/hoon:<[172 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 16].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 20].[173 27]>
/~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
crud: %reap event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[171 3].[174 49]>
[%drum-unauthorized [%our ~zod] %src ~zod]
/~zod/home/0/lib/hood/drum/hoon:<[172 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 16].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 20].[173 27]>
/~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
crud: %reap event failed
/~zod/home/0/app/hood/hoon:<[118 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 3].[119 20]>
/~zod/home/0/app/hood/hoon:<[119 9].[119 19]>
/~zod/home/0/lib/hood/drum/hoon:<[171 3].[174 49]>
[%drum-unauthorized [%our ~zod] %src ~zod]
/~zod/home/0/lib/hood/drum/hoon:<[172 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 3].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 16].[174 49]>
/~zod/home/0/lib/hood/drum/hoon:<[173 20].[173 27]>
/~zod/home/0/lib/hood/drum/hoon:<[497 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 3].[500 35]>
/~zod/home/0/lib/hood/drum/hoon:<[498 7].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[498 11].[498 18]>
/~zod/home/0/lib/hood/drum/hoon:<[351 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[352 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[353 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 3].[354 36]>
/~zod/home/0/lib/hood/drum/hoon:<[354 4].[354 36]>
~zod:dojo>

</p>
<details>

**Comment by yosoyubik** [comment]
Closing this since using the latest version from master doesn't cause the aforementioned error.

**Comment by yosoyubik** [comment]
Reopening since I've experienced this again (and found a previous issue that seems to be related: https://github.com/urbit/arvo/issues/294)

**Comment by juaningan** [comment]
Im getting same behavior after `|unlink %chat-cli`

**Comment by joemfb** [comment]
@Fang-, do you think #4167 fixed this?

**Comment by Fang-** [comment]
Considering this seems to be a duplicate of #1790, yes, I would think so!
